src/common/codeUtil.ts

```diff
@@ -8,12 +8,13 @@ export class CodeUtil {
             return '';
         }
         return original
-            .replace(/;.+/, '')
             .replace(/".*?"/g, '""') // replace string literals with empty string literal
             .replace(/{.*}/g, '') // remove matching braces
             .replace(/ +/g, ' ')
             .replace(/\bgui\b.*/gi, '')
-            .replace(/\b(msgbox)\b.+?%/gi, '$1');
+            .replace(/\b(msgbox)\b.+?%/gi, '$1')
+            .replace(/;.+/, '') // remove comments must be last, semicolon may be inside string (expression) or 'MsgBox, { ; comment with close brace }'
```

Comment:
Can we add some unit tests for `purify`? We can add them in `extension.test.ts` for now.

Reply:
I revert this changes, this function modified in #159